src/api/apps/articles/model/index.js

```diff
@@ -289,6 +294,18 @@ export const backfill = (callback) => {
         Write backfill logic here. Make sure to callback with cb()
         eg: distributeArticle(article,cb)
       */
+
+      const url = getArticleUrl(article)
+
+      const deleteCb = () => {
+        console.log('article deleted')
+      }
+
+      // remove existing article from sailthru
+      deleteArticleFromSailthru(url, deleteCb)
```

Comment:
Do we need this? Isn't this covered in `cleanArticlesFromSailthru` already in distribute?

Reply:
I believe cleanArticlesFromSailthru deletes all but the last article.  Added this on the assumption that we want to delete all existing articles and resend with new args.

Reply:
Oh is this to cover the case of /video and /series? If we delete the article entirely, i wonder if it would remove analytics on it too? It might be worth checking the docs

Reply:
Usually you can update an article by passing in the same url and giving it new args

Reply:
I'm not very familiar with analytics related to email, which docs do you mean?  Is this internal artsy or Sailthru specific?